packages/showcases/svelte/stories/components/navigation/VtmnNavbar/VtmnNavbar.stories.svelte

```diff
@@ -50,7 +50,11 @@
   <div style="width: 800px; display: flex; justify-content: center">
     <VtmnNavbar {...args}>
       <svelte:fragment slot="left-nav">
-        <VtmnButton iconAlone="menu-line" variant="ghost" />
+        <VtmnButton
+          iconAlone="menu-line"
+          variant="ghost"
+          aria-expanded="false"
```

Comment:
The expanded is not always false. So we can't add this like this.
@GaspardMathon I let you check with @Tlahey 👌🏻

Reply:
It's just for the example. It's like some components examples with an `aria-disabled` set to true. So I don't know what should I do ? Remove it ?